### DISCUSSION TITLE
I dont know how to use it

### DISCUSSION THREAD

**Discussion by ChirinoFrancisco** [sigma67/spotify_to_ytmusic#108] [Q&A]
Can someone explain me how to use this tool? Im not so use to Python

**Comment by sctech-tr** [comment]
there is documentation for that in the readme. please read that.
you don't even have to deal with python, you just run a single command.

**Comment by golfhou** [comment]
This is what I did, and you could do in almost any OS.

Create a debian Virtual Machine.
if running as root when you login:

1. apt update && apt upgrade -y
2. apt install \
      python3 \
      python3-pip \
      python3-venv \
      
pip install pipx

3. pipx ensurepath

Now you have to open a new shell.  Since I was on Proxmox, I edited /etc/ssh/sshd_config using nano to allow root on SSH, that was disabled.

on new shell window, i used terminal on Mac OS.

4. spotify_to_ytmusic setup
5. You are asked to setup API for both Spotify and Youtube.  You have to get your spotify client id and secret from the spotify developer dashboard. 
6. You are then given a google link to setup with your google account so the program creates the playlist on youtube.
7. once both are setup, spotify_to_ytmusic create <link of spotify playlist> and that is it
8. "Success: created playlist" <link to the youtube playlist> - this will be private playlist.

You can then delete the virtual machine if you no longer need to transfer spotify playlists.

Great, program!
